velox/dwio/dwrf/reader/StripeStream.h

```diff
@@ -105,6 +105,15 @@ class StripeStreams {
    */
   virtual const dwio::common::ColumnSelector& getColumnSelector() const = 0;
 
+  /**
+   * Session timezone used for reading Timestamp.
+   */
+  virtual const tz::TimeZone* getSessionTimezone() const = 0;
+
+  virtual bool adjustTimestampToTimezone() const {
```

Comment:
Just leave this as pure virtual as well?

Reply:
Refactored